DuckDuckGo/SettingsSubscriptionView.swift

```diff
@@ -87,148 +61,157 @@ struct SettingsSubscriptionView: View {
+        })
+            .environmentObject(subscriptionNavigationCoordinator)
+        NavigationLink(destination: settingsView) {
+            SettingsCellView(
+                label: UserText.settingsPProManageSubscription,
+                subtitle: UserText.settingsPProSubscriptionExpiredTitle,
+                image: Image("SettingsPrivacyPro"),
+                accessory: .image(Image("Exclamation-Color-16"))
+            )
         }
     }
-    
+
     @ViewBuilder
     private var noEntitlementsAvailableView: some View {
-        Group {
-            SettingsCustomCell(content: {
-                HStack(alignment: .top) {
-                    Image(Constants.infoIcon)
-                        .frame(width: Constants.noEntitlementsIconWidth)
-                        .padding(.top, Constants.topCellPadding)
-                    VStack(alignment: .leading) {
-                        Text(UserText.settingsPProActivationPendingTitle).daxBodyRegular()
-                        Text(UserText.settingsPProActivationPendingDescription).daxFootnoteRegular()
-                            .padding(.bottom, Constants.purchaseDescriptionPadding)
-                    }.foregroundColor(Color(designSystemColor: .textSecondary))
-                }
-            })
-            restorePurchaseView
+        disabledFeaturesView
+        
+        // Renew Subscription (Expired)
+        let settingsView = SubscriptionSettingsView(configuration: .activating,
+                                                    settingsViewModel: viewModel,
+                                                    viewPlans: {
+            subscriptionNavigationCoordinator.shouldPushSubscriptionWebView = true
+        })
+            .environmentObject(subscriptionNavigationCoordinator)
+        NavigationLink(destination: settingsView) {
+            SettingsCellView(
+                label: UserText.settingsPProManageSubscription,
+                subtitle: UserText.settingsPProActivating,
+                image: Image("SettingsPrivacyPro")
+            )
         }
     }
-    
+
     @ViewBuilder
     private var subscriptionDetailsView: some View {
         
         if viewModel.state.subscription.entitlements.contains(.networkProtection) {
-            SettingsCellView(
-                label: UserText.settingsPProVPNTitle,
-                subtitle: viewModel.state.networkProtection.status != ""
-                ? viewModel.state.networkProtection.status : nil,
-                action: { viewModel.presentLegacyView(.netP) },
-                disclosureIndicator: true,
-                isButton: true)
+            SettingsCellView(label: UserText.settingsPProVPNTitle,
+                             image: Image("SettingsPrivacyProVPN"),
+                             action: { viewModel.presentLegacyView(.netP) },
+                             statusIndicator: StatusIndicatorView(status: viewModel.state.networkProtection.enabled ? .on : .off),
```

Comment:
You use `viewModel.state.networkProtection.enabled` but it is never set anywhere in the code. 
Please see `func updateNetPStatus(connectionStatus: ConnectionStatus)` in `SettingsViewModel` (there is only `status` string set there which is obsolete.

Reply:
Also if you set `viewModel.state.networkProtection.enabled` to `true` you will get an additional `VPN` cell in the "Privacy Protections" section of the settings.

Reply:
Fixed, I'm creating a new build